### ISSUE TITLE
[Feature Request] GIT-LFS Image Transformations in Dev

### ISSUE THREAD

**Issue by aldendaniels** [netlify/cli#668]
**TL;DR:** It would be terrific if [image transformation feature](https://www.netlify.com/blog/2019/02/26/manage-your-code-and-assets-together-with-netlify-large-media/) worked via netlify dev. 

(Originally posed to the [forum](https://community.netlify.com/t/git-lfs-image-transformations-in-dev/6537) but posting here as suggested by @fool.)

Scenario: I'm using Netlify's [image transformation feature](https://www.netlify.com/blog/2019/02/26/manage-your-code-and-assets-together-with-netlify-large-media/) to transform images uploaded via `git-lfs` for a static website. I can test all aspects of the website locally except for image resizing. To test this, I have to push to the CI as the cropping arguments are ignored in dev.

**Comment by TigersWay** [comment]
+1
Would also love to see such a feature... Have been tinkering around that and while working it's still inefficient.

**Comment by byebyers** [comment]
Yes being able to test this in development would be really helpful.

**Comment by sarahetter** [comment]
Large media is becoming deprecated.
